tdrs-backend/tdpservice/settings/common.py

```diff
@@ -84,7 +84,9 @@ class Common(Configuration):
     ROOT_URLCONF = "tdpservice.urls"
     SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
     WSGI_APPLICATION = "tdpservice.wsgi.application"
-    CORS_ORIGIN_ALLOW_ALL = True
+    
+    # CORS allowed origins
+    CORS_ALLOWED_ORIGINS = ['https://tanfdata.acf.hhs.gov']
```

Comment:
Should we only want this for `tanf-prod`?

Reply:
totally right: should be moved to cloudgov